l/logstash/build_info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "package_name" : "logstash",
+    "github_url":"https://github.com/elastic/logstash.git",
+    "package_dir": "l/logstash/Dockerfiles/",
```

Comment:
@muskaan2199  this should be path to build script and not dockerfile.

Also keep build script name added for default case so that if regex pattern not matched with `v8.3.*` or `v8.4.*`, it will still use latest script to validate for future versions.